Sources/Architecture/X86/HardwareBreakpointManager.cpp

```diff
@@ -48,17 +48,33 @@ ErrorCode HardwareBreakpointManager::add(Address const &address, Type type,
   return super::add(address, type, size, mode);
 }
 
+ErrorCode HardwareBreakpointManager::remove(Address const &address) {
+  auto loc = std::find(_locations.begin(), _locations.end(), address);
+  if (loc != _locations.end()) {
+    _locations[loc - _locations.begin()] = 0;
+  }
+
+  return super::remove(address);
```

Comment:
If `loc == _locations.end()`, does it make sense to call `super::remove(address)`? I'd probably do 
```
if (loc == _locations.end()) {
    return kErrorSomething;
}

_locations[...] = 0;
return super::remove(address);
```